database_scripts/004_add_hei_access.sql

```diff
@@ -0,0 +1,32 @@
+BEGIN TRAN AdditionalPolicy
+ DECLARE @policyid UNIQUEIDENTIFIER
+ DECLARE @covUniOrgId UNIQUEIDENTIFIER
+ DECLARE @boltonUniOrgId UNIQUEIDENTIFIER
+ DECLARE @bathUniOrgId UNIQUEIDENTIFIER
+ DECLARE @teacherITTServiceId UNIQUEIDENTIFIER
+ DECLARE @teacherEmpServiceId UNIQUEIDENTIFIER
+ DECLARE @evolveEmpMatPolicyId UNIQUEIDENTIFIER
+ DECLARE @evolveEmpPolicyId UNIQUEIDENTIFIER
+ SET @policyid = newid();
+ SET @teacherITTServiceId = (select top 1 id from [service] where name = 'Teacher Services - ITT Provider');
+ SET @covUniOrgId = (select top 1 Id from [organisation] where UKPRN = '10001726' and URN = '133808');
```

Comment:
can we be consistent in our syntax writing, any SQL keyword should be in uppercase